packages/expo-image/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unpublished
 
+- Update `ImageProps` type so `children` are omitted. ([#33210](https://github.com/expo/expo/pull/33210) by [@ashaller2017](https://github.com/ashaller2017))
```

Comment:
It's still in a wrong place. Can you please move it to "Others" category?

Reply:
Read previous comment wrong. Apologies.